hazelcast/src/main/java/com/hazelcast/map/impl/record/AbstractRecordWithStats.java

```diff
@@ -26,11 +24,8 @@
     protected RecordStatistics recordStatistics;
 
     protected AbstractRecordWithStats() {
-    }
-
-    protected AbstractRecordWithStats(Data key) {
-        super(key);
-        this.recordStatistics = new RecordStatistics();
+        super();
```

Comment:
There is no need to call super. If you do not provide one, one is emitted by the Java Compiler. So this is only noise and can be removed.